markdown-pages/zh/terms/licenses.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 社区软件许可协议
+summary: ''
+---
+
+# PingCAP 社区软件许可协议
+
+**下载或使用本协议随附的软件前，请您仔细阅读下述条款和条件（特别是粗体标注内容）。使用软件或点击“我接受 (I ACCEPT)”按钮或类似的按钮，即表示您同意仅在遵循下述条款和条件的前提下使用本协议随附的软件。**
+
+本 PingCAP 社区软件许可协议（下称“本协议”）系为您与北京平凯星辰科技发展有限公司（下称“PingCAP”，PingCAP 与其关联方以下合称“PingCAP 集团”）之间具有法律约束力的合同，并约束您对 PingCAP 社区软件（定义如下）的使用。“PingCAP 社区软件”，系指由 PingCAP 集团提供的社区版软件，包括但不限于 TiDB 数据库（社区版），TiFlash 列存引擎（社区版），TiSpark 计算引擎（社区版），TiDB 周边运维工具（社区版）以及任何 PingCAP 社区服务或前述软件的更新，以及本协议第 1 条所述的 PingCAP 社区软件的所有版本。
+
+**通过安装或以其他方式使用 PingCAP 社区软件或点击“我接受 (I ACCEPT)” 按钮，即表示您同意本协议的条款并同意于安装、使用 PingCAP 社区软件之日或点击“我接受 (I ACCEPT)” 按钮之日（以孰早者为准）受本协议约束。**
+
+**您保证您至少年满十八周岁，并且您拥有签署合同的法定能力。如果您代表任何公司、组织或其他实体签署本协议，则 (a) 本协议中所称“您”同时亦指该公司、组织、实体或其关联方； (b) 您声明并保证您拥有约束该公司、组织、实体或其关联方接受本协议所规定条款和条件的授权；（c）您无需取得其他第三方授权方可签署并履行本协议；并且（d）签署和履行本协议不会违反您已签署的其他协议或适用的法律法规。如果您不同意本协议的任何条款或尚未取得授权签署本协议，则必须立即停止使用 PingCAP 社区软件，取消并删除所有副本。**
+
+1. **协议的适用版本。** 如果您已按照任何其他许可条款安装或使用了 PingCAP 社区软件的任何先前版本，则通过安装或使用此协议随附的社区软件的版本，即表示您同意本协议将全部替换那些其他许可条款并且还将管辖您对 PingCAP 社区软件的所有先前版本的使用。
+
+2. **期限和终止。** 本协议将持续适用，直至（a）您随时通过卸载和删除您下载或控制的 PingCAP 社区软件的所有副本来终止本协议；或（b）PingCAP 终止本协议。PingCAP 可能会出于任何原因（包括但不限于您违反本协议）随时通过向您发出通知来终止本协议。如果 PingCAP 通知您终止，您必须卸载并删除您拥有或控制的 PingCAP 社区软件的所有副本。
+
+3. **协议的修改。如 PingCAP 需要对本协议进行变更或修改的，PingCAP 会通过本公司网站 https://pingcap.com/licenses/，通过网站公告的方式提前予以公布；若您在本协议内容公告变更后继续使用PingCAP社区软件的，表示您已充分阅读、理解并接受变更后的协议内容，也将遵循变更后的协议内容。** PingCAP 集团也可能会不时通过 PingCAP 社区软件的任何更新或新版本通过通知您的方式修改本协议或以新条款替换本协议。
```

Comment:
Should have a whitespace after the link, or the following text will be included. Or use a markdown link perhaps.